### PR TITLE
react-native 0.25.1 compatibility

### PR DESCRIPTION
**Pull request by PyYoshi**

### CODE REVIEW

File: lib/internal/Thumb.js [hunk 1]
```diff
@@ -8,15 +8,19 @@
...
+} = React;
 
 const {
-  Component,
   Animated,
   View,
   PanResponder,
-  PropTypes,
-} = React;
+  PropTypes
```
Comment (jeduan) [inline lib/internal/Thumb.js]: PropTypes should be imported from React

**Comment by PyYoshi** [comment]
@jeduan thank you!

**Comment by Daniel-ltw** [comment]
Button.js:157 will break. It should be ReactNative.Text

**Comment by PyYoshi** [comment]
@Daniel-ltw thank you!

**Comment by Daniel-ltw** [comment]
@PyYoshi , Do you have problems with the buttons? 
Mine just has a red border and would not work.

**Comment by PyYoshi** [comment]
@Daniel-ltw

I confirmed on Android).

But iOS Simulator has no problem.

Android (Nexus 5 6.0.1): 

Android

iOS Simulator (iPhone 6/ iOS 9.3): 

iOS

**Comment by Daniel-ltw** [comment]
No issues on mine. Just need a re-build. 

Cheers mate.

**Comment by mcrumm** [comment]
Any chance of seeing this merged soon?  `0.26.x` removes the deprecation warnings for using `React` et. al from `react-native` and just straight breaks.

**Comment by PyYoshi** [comment]
@xinthink Please check this PR?

**Comment by xinthink** [comment]
Yes, of course, I'll look into it ASAP. Got a little busy for a while :tired_face:

**Comment by PyYoshi** [comment]
@xinthink Thank you!

**Comment by mcrumm** [comment]
@xinthink Thanks!  Could you tag a new release, too?

**Comment by ajoshdee** [comment]
Hi, how can i use the merged version? I keep getting the old version

**Comment by jwarby** [comment]
@ajoshdee You can install the master branch from GitHub until a new release is publishing by doing:

```bash
npm install --save xinthink/react-native-material-kit
```

**Pull request merged**